src/widgets.js

```diff
@@ -653,6 +657,14 @@ define([
             });
         };
 
+        if (options.defaultOptions) {
+            widget.addOptions(options.defaultOptions);
+
+            if (options.defaultValue) {
```

Comment:
Not quite following why this was added...should the spec in mugs.js include a defaultValue?